etna/media/forms.py

```diff
@@ -0,0 +1,64 @@
+import magic
+
+from django import forms
+from django.core.exceptions import ValidationError
+
+from wagtail.admin import widgets
+
+from wagtailmedia.permissions import permission_policy as media_permission_policy
+from wagtail.admin.forms.collections import BaseCollectionMemberForm, collection_member_permission_formset_factory
```

Comment:
Looks like there's an unused import for `collection_member_permission_formset_factory`

Reply:
Thanks. I've removed it now.